### ISSUE TITLE
expireAfterSeconds schema option not valid and not effective on timeseries collection

### ISSUE THREAD

**Issue by twinwise73** [Automattic/mongoose#11229]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
I use moongose for create timeseries collection
my schema is like this : 
```
const schema = new mongoose.Schema({ timestamp: Date, metadata: metricMetadataSchema },
  {
    _id: false,
    strict: false,
    autoIndex: false,
    autoCreate: false,
    timeseries: {timeField: "timestamp", metaField: "metadata", granularity: 'minutes'},
    // @ts-ignore
    expireAfterSeconds: 60
  }
)
```
At this point the property expireAfterSeconds is not valid for typescript because expireAfterSeconds not include in type SchemaOptions
The collection is well  done created, but no info about expireAfterSeconds anywhere  and after 60 seconds data not removed.

**If the current behavior is a bug, please provide the steps to reproduce.**

```
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "es2017",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "skipLibCheck": true
  }
}
```
**What is the expected behavior?**

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**
node: v14.17.0
mongodb: 4.3.0
mongoose: 6.1.6

**Comment by Uzlopak** [comment]
Well... expireAfterSeconds has to be in the attribute timeseries and not somewhere else.
see https://docs.mongodb.com/manual/core/timeseries-collections/

So it should be

const schema = new mongoose.Schema({ timestamp: Date, metadata: metricMetadataSchema },
  {
    _id: false,
    strict: false,
    autoIndex: false,
    autoCreate: false,
    timeseries: {
        timeField: "timestamp", 
        metaField: "metadata",
        granularity: 'minutes',
        expireAfterSeconds: 60
        },
  }
)

**Comment by Uzlopak** [comment]
Ok, should be fixed with #11489
